app/lib/backend.rb

```diff
@@ -0,0 +1,43 @@
+class Backend
+  include HTTParty
+  base_uri Settings.backend.url
+
+  def version
+    handle_response { self.class.get("/api/version") }
+  end
+
+  def bots
```

Comment:
httparty is designed to use class methods as api calls and not instance methods. There should be no `Backend.new`